CONTRIBUTING.md

```diff
@@ -136,4 +136,4 @@ If you add or update configuration options, regenerate and commit the documentat
 
 ## More information
 
-For more information, including details about contributing to the code as well as the documentation for LXD, see [Contributing](https://documentation.ubuntu.com/lxd/en/latest/contributing/) in the documentation.
\ No newline at end of file
+For more information, including details about contributing to the code as well as the documentation for LXD, see [How to contribute to LXD](https://documentation.ubuntu.com/lxd/en/latest/contributing/) in the documentation.
```

Comment:
@minaelee please can you backport this change into the stable-5.21 branch but with links to the 5.21 version of the docs?